### PR TITLE
Update ic-js version to 2023-08-09

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Update ic-js version to add `disburseMaturity`

# Changes

-  `npm run update:next`
- update sns aggregator types

# Tests

- current tests works.

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

**Review by lmuntaner — APPROVED** [review]
Thanks!

File: frontend/src/tests/mocks/sns-response.mock.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ import {
 const swapToQuerySwap = (swap: SnsSummarySwap): [SnsSwap] => [
   {
     ...swap,
+    already_tried_to_auto_finalize: swap.already_tried_to_auto_finalize,
```
Comment (lmuntaner) [inline frontend/src/tests/mocks/sns-response.mock.ts:28]: Is this needed?

File: frontend/src/lib/types/sns.ts [hunk 2]
```diff
@@ -25,6 +26,11 @@ export interface SnsSummaryMetadata {
 }
 
 export interface SnsSummarySwap {
+  auto_finalize_swap_response: [] | [FinalizeSwapResponse];
+  next_ticket_id?: bigint;
+  already_tried_to_auto_finalize: [] | [boolean];
+  purge_old_tickets_last_completion_timestamp_nanoseconds?: bigint;
```
Comment (lmuntaner) [inline frontend/src/lib/types/sns.ts:32]: Why not `[] | [bigint]`?

File: frontend/src/lib/types/sns.ts [hunk 3]
```diff
@@ -25,6 +26,11 @@ export interface SnsSummaryMetadata {
 }
 
 export interface SnsSummarySwap {
+  auto_finalize_swap_response: [] | [FinalizeSwapResponse];
+  next_ticket_id?: bigint;
```
Comment (lmuntaner) [inline frontend/src/lib/types/sns.ts:30]: Why not `[] | [bigint]`

Comment (mstrasinskis) [reply frontend/src/lib/types/sns.ts]: Thanks for the explanation.

File: frontend/src/lib/types/sns.ts
Comment (lmuntaner) [inline frontend/src/lib/types/sns.ts:1]: Adding this comment to remind you of adding an entry in the Changelog.

**Review by lmuntaner — APPROVED** [review]
LGTM!

Just one open comment to add an entry in the changelog. Approving already.

**Pull request merged**